depends/patches/fontconfig/gperf_header_regen.patch

```diff
@@ -0,0 +1,24 @@
+commit 8a4c858f1c2e95c4e2483f9d5852b4a61158f3f2
+Author: fanquake <fanquake@gmail.com>
+Date:   Mon Aug 17 12:50:35 2020 +0000
+
+    Remove rule that causes inadvertant header regeneration
+
+    Otherwise the makefile will needlessly attempt to re-generate the
+    headers with gperf. This can be dropped once the upstream build is fixed.
+
+    See #10851.
+
+diff --git a/src/Makefile b/src/Makefile
+index aa9ea57..ded34a8 100644
+--- a/src/Makefile
++++ b/src/Makefile
```

Comment:
45f33606377cda7065fe30aa4eeee5033e9b0699

I suggest to apply this patch to `Makefile.in` and move all patching work into the `preprocess_cmds` stage.
Patching the source file rather the generated one is simpler to verify and maintain, no?

Reply:
I've moved the patching to preprocessing.